odoo/signals.py

```
@@ -0,0 +1,123 @@
+    if instance.status in [
+        InvoiceStatusOptions.issued.value,
+    ]:
+        odoo_integration = OdooInvoiceResource()
+        odoo_integration.sync_invoice_to_odoo_api(instance.external_id)
+    elif instance.status in INVOICE_CANCELLED_STATUS:
+        odoo_integration = OdooInvoiceResource()
+        odoo_integration.sync_invoice_return_to_odoo_api(instance.external_id)
+
+
+@receiver(post_save, sender=PaymentReconciliation)
+def sync_payment_to_odoo(sender, instance, created, **kwargs):
```

Ruff (0.14.3)

Unused function argument: `sender`

(ARG001)